lib/src/dlna/discovery.rs

```diff
@@ -0,0 +1,38 @@
+use rupnp::{Device, ssdp::URN};
+use std::{error::Error, time::Duration};
+use futures_util::StreamExt;
+use crate::dlna::models::DlnaDevice;
+
+const EXTRA: &[&str; 1] = &["roomName"];
+
+pub  async fn discover_devices() -> Result<Vec<DlnaDevice>, Box<dyn Error>> {
+    let mut dlna_devices = Vec::new();
+    
+    let search_target = URN::device("schema-upnp-org", "MediaServer", 1).into();
```

Comment:
```suggestion
    let search_target = URN::device("schemas-upnp-org", "MediaServer", 1).into();
```

From my basic knowledge of DLNA, this should be `schemas` instead of `schema`.
This way it actually works and finds my jellyfin DLNA server.

Reply:
I did not find the typo, so I copied the already running line from my PoC. But thanks.